sbin/Makefile.am

```diff
@@ -9,20 +7,19 @@ scriptsin = \
 	update-ipsets.in \
 	vnetbuild.in
 
-inclibdir = $(libarchinddir)/firehol
+inclibdir = $(libexecdir)/firehol/@PACKAGE_VERSION@
 
 SUFFIXES = .in
 .in:
 	if [ "$@" = "commands.sed.in" ]; then \
 		true; \
 	elif sed \
-		-e '/^# Start defaults before configure/,/^# End/d' \
+		-e 's#MAKE_RUN#Y#g' \
 		-e 's#[$$]prefix_POST#$(prefix)#g' \
 		-e 's#[$$]bindir_POST#$(bindir)#g' \
 		-e 's#[$$]libdir_POST#$(inclibdir)#g' \
 		-e 's#[$$]localstatedir_POST#$(localstatedir)#g' \
 		-e 's#[$$]sysconfdir_POST#$(sysconfdir)#g' \
-		-f commands.sed \
```

Comment:
can you please move this to $(top_srcdir)/build/subst.inc so we can use it in multiple makefiles? see recent netdata change.